pkg/apis/configuration/validation/policy.go

```diff
@@ -284,6 +302,51 @@ func validateLogConf(logConf, logDest string, fieldPath *field.Path) field.Error
 	return allErrs
 }
 
+func validateBados(bados *v1.Bados, fieldPath *field.Path) field.ErrorList {
```

Comment:
Could we have tests similar to TestValidateWAF and TestValidateWAFInvalid?

Reply:
Done